src/main/java/com/google/cloud/flink/bigquery/StandardSQLTypeHandler.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.flink.bigquery;
+
+import com.google.cloud.bigquery.StandardSQLTypeName;
+import org.apache.flink.table.types.logical.LogicalType;
+
+public class StandardSQLTypeHandler {
+
+  public static StandardSQLTypeName handle(LogicalType sqlType) {
```

Comment:
Why not to create a `Map<LogicalTypeRoot, StandardSQLTypeName>` ? It saves the toString and the switch.

Reply:
addressed